plugins/available/docker-machine.bash

```diff
@@ -0,0 +1,11 @@
+cite about-plugin
+about-plugin 'Helpers to get Docker setup correctly for docker-machine'
+
+# Note, this might need to be different if you use a machine other than 'dev',
+# or its configured for a different IP
+if [[ `uname -s` == "Darwin" ]]; then
+  export DOCKER_HOST="tcp://192.168.99.100:2376"
```

Comment:
Yes, that's right. I have been using the same static IP with boot2docker for a while now and it has not changed.